File: Programmers/Lv0/짝수홀수개수.py
def solution(num_list):
    even = 0
    odd = 0
    answer = []
    for num in num_list:
        if (num % 2 == 1):
            odd += 1
        else:
            even += 1
    answer.append(even)
    answer.append(odd)
    return answer

# 짝수자리에 짝수 넣고, 홀수자리에 홀수 넣는 방법
def solution(num_list):
    answer = [0,0]
    for n in num_list:
        answer[n%2]+=1
    return answer

File: Programmers/Lv0/짝수홀수개수.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer;
    int even = 0;
    int odd = 0;
    for (int num : num_list)
        num % 2 == 0 ? ++even : ++odd;
    answer.push_back(even);
    answer.push_back(odd);
    return answer;
}

// vector의 생성자를 이용한 방법 1
vector<int> solution(vector<int> num_list) {
    vector<int> answer(2, 0);
    for (int num : num_list) {
        answer[num % 2]++;
    }
    return answer;
}
